Update examples to use latest version of the charting library

```
ERROR in src/app/tv-chart-container/tv-chart-container.component.ts(4,5): error TS2305: Module '"C:/Users/mitja/Downloads/charting-library-examples-3fabf861de413f8ca4a32ad0df72ea6c2a0909a0/angular5/src/assets/charting_library/charting_library.min"' has no exported member 'onready'.
src/app/tv-chart-container/tv-chart-container.component.ts(113,22): error TS2339: Property 'attr' does not exist on type 'HTMLElement'.
```

Can someone help what to fix?

--- a/angular5/src/app/tv-chart-container/tv-chart-container.component.ts
+++ b/angular5/src/app/tv-chart-container/tv-chart-container.component.ts
@@ -110,18 +110,19 @@ export class TvChartContainerComponent implements OnInit, OnDestroy {
         this._tvWidget = tvWidget;
 
         tvWidget.onChartReady(() => {
-            const button = tvWidget.createButton()
-                .attr('title', 'Click to show a notification popup')
-                .addClass('apply-common-tooltip')
-                .on('click', () => tvWidget.showNoticeDialog({
-                    title: 'Notification',
-                    body: 'TradingView Charting Library API works correctly',
-                    callback: () => {
-                        console.log('Noticed!');
-                    },
-                }));
-
-            button[0].innerHTML = 'Check API';
+            tvWidget.headerReady().then(() => {
+                const button = tvWidget.createButton();
+                button.setAttribute('title', 'Click to show a notification popup');
+                button.classList.add('apply-common-tooltip');
+                button.addEventListener('click', () => tvWidget.showNoticeDialog({
+                        title: 'Notification',
+                        body: 'TradingView Charting Library API works correctly',
+                        callback: () => {
+                            console.log('Noticed!');
+                        },
+                    }));
+                button.innerHTML = 'Check API';
+            });
         });
     }
 

--- a/react-javascript/src/components/TVChartContainer/index.jsx
+++ b/react-javascript/src/components/TVChartContainer/index.jsx
@@ -51,10 +51,11 @@ export class TVChartContainer extends React.PureComponent {
 		this.tvWidget = tvWidget;
 
 		tvWidget.onChartReady(() => {
-			const button = tvWidget.createButton()
-				.attr('title', 'Click to show a notification popup')
-				.addClass('apply-common-tooltip')
-				.on('click', () => tvWidget.showNoticeDialog({
+			tvWidget.headerReady().then(() => {
+				const button = tvWidget.createButton();
+				button.setAttribute('title', 'Click to show a notification popup');
+				button.classList.add('apply-common-tooltip');
+				button.addEventListener('click', () => tvWidget.showNoticeDialog({
 					title: 'Notification',
 					body: 'TradingView Charting Library API works correctly',
 					callback: () => {
@@ -62,7 +63,8 @@ export class TVChartContainer extends React.PureComponent {
 					},
 				}));
 
-			button[0].innerHTML = 'Check API';
+				button.innerHTML = 'Check API';
+			});
 		});
 	}
 

--- a/react-typescript/package.json
+++ b/react-typescript/package.json
@@ -13,11 +13,12 @@
     "test": "react-scripts-ts test"
   },
   "devDependencies": {
+    "@types/jest": "^24.0.16",
     "@types/jquery": "^1.10.31",
     "@types/react": "^16.0.35",
     "@types/react-dom": "^16.0.3",
     "tslint": "~5.9.1",
     "tslint-react": "^3.5.1",
-    "typescript": "^2.6.2"
+    "typescript": "^3.5.2"
   }
 }

--- a/react-typescript/src/components/TVChartContainer/index.tsx
+++ b/react-typescript/src/components/TVChartContainer/index.tsx
@@ -77,17 +77,19 @@ export class TVChartContainer extends React.PureComponent<Partial<ChartContainer
 		this.tvWidget = tvWidget;
 
 		tvWidget.onChartReady(() => {
-			const button = tvWidget.createButton();
-			button.setAttribute('title', 'Click to show a notification popup');
-			button.setAttribute('class', 'apply-common-tooltip');
-			button.addEventListener('click', () => tvWidget.showNoticeDialog({
-					title: 'Notification',
-					body: 'TradingView Charting Library API works correctly',
-					callback: () => {
-						console.log('Noticed!');
-					},
-				}));
-			button.innerHTML = 'Check API';
+			tvWidget.headerReady().then(() => {
+				const button = tvWidget.createButton();
+				button.setAttribute('title', 'Click to show a notification popup');
+				button.classList.add('apply-common-tooltip');
+				button.addEventListener('click', () => tvWidget.showNoticeDialog({
+						title: 'Notification',
+						body: 'TradingView Charting Library API works correctly',
+						callback: () => {
+							console.log('Noticed!');
+						},
+					}));
+				button.innerHTML = 'Check API';
+			});
 		});
 	}
 

--- a/ruby-on-rails/app/assets/javascripts/tv-chart-widget-init.js
+++ b/ruby-on-rails/app/assets/javascripts/tv-chart-widget-init.js
@@ -5,7 +5,7 @@ function getLanguageFromURL() {
 	return results === null ? null : decodeURIComponent(results[1].replace(/\+/g, ' '));
 }
 
-TradingView.onready(function() {
+function initOnReady() {
 	var widget = window.tvWidget = new TradingView.widget({
 			symbol: 'AAPL',
 			// BEWARE: no trailing slash is expected in feed URL
@@ -28,10 +28,13 @@ TradingView.onready(function() {
 	});
 
 	widget.onChartReady(() => {
-		const button = widget.createButton()
-			.attr('title', 'Click to show a notification popup')
-			.addClass('apply-common-tooltip')
-			.on('click', () => widget.showNoticeDialog({
+		widget.headerReady().then(() => {
+			const button = widget.createButton();
+
+			button.setAttribute('title', 'Click to show a notification popup');
+			button.classList.add('apply-common-tooltip');
+
+			button.addEventListener('click', () => widget.showNoticeDialog({
 				title: 'Notification',
 				body: 'TradingView Charting Library API works correctly',
 				callback: () => {
@@ -39,6 +42,9 @@ TradingView.onready(function() {
 				},
 			}));
 
-		button[0].innerHTML = 'Check API';
+			button.innerHTML = 'Check API';
+		});
 	});
-});
+};
+
+window.addEventListener('DOMContentLoaded', initOnReady, false);

--- a/vuejs/babel.config.js
+++ b/vuejs/babel.config.js
@@ -1,5 +1,8 @@
 module.exports = {
-  presets: [
-    '@vue/app'
-  ]
+    presets: [
+        '@vue/app',
+    ],
+    plugins: [
+        '@babel/plugin-transform-modules-commonjs'
+    ],
 }

--- a/vuejs/package.json
+++ b/vuejs/package.json
@@ -11,9 +11,9 @@
     "vue": "~2.5.17"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "3.0.4",
+    "@vue/cli-plugin-babel": "^3.9.2",
     "@vue/cli-plugin-eslint": "3.0.4",
-    "@vue/cli-service": "3.0.4",
+    "@vue/cli-service": "^3.9.3",
     "babel-core": "6.26.3",
     "babel-loader": "7.1.5",
     "node-sass": "~4.9.4",

--- a/vuejs/src/components/TVChartContainer.vue
+++ b/vuejs/src/components/TVChartContainer.vue
@@ -89,19 +89,23 @@ export default {
     this.tvWidget = tvWidget;
 
     tvWidget.onChartReady(() => {
-      const button = tvWidget.createButton()
-        .attr('title', 'Click to show a notification popup')
-        .addClass('apply-common-tooltip')
-        .on('click', () => tvWidget.showNoticeDialog({
-          title: 'Notification',
-          body: 'TradingView Charting Library API works correctly',
-          callback: () => {
-            // eslint-disable-next-line no-console
-            console.log('Noticed!');
-          },
-        }));
+      tvWidget.headerReady().then(() => {
+        const button = tvWidget.createButton();
 
-      button[0].innerHTML = 'Check API';
+        button.setAttribute('title', 'Click to show a notification popup');
+        button.classList.add('apply-common-tooltip');
+
+        button.addEventListener('click', () => tvWidget.showNoticeDialog({
+            title: 'Notification',
+            body: 'TradingView Charting Library API works correctly',
+            callback: () => {
+              // eslint-disable-next-line no-console
+              console.log('Noticed!');
+            },
+          }));
+
+        button.innerHTML = 'Check API';
+      });
     });
   },
   destroyed() {

